rpc/lib/server/http_server.go

```diff
@@ -56,7 +56,7 @@ func WriteRPCResponseHTTPError(w http.ResponseWriter, httpCode int, res types.RP
 
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(httpCode)
-	w.Write(jsonBytes)
+	_, _ = w.Write(jsonBytes) // ignoring error
```

Comment:
why `ignoring error` vs `nolint` ?

Reply:
nolint is good for the defers where it's a hassle to acknowledge the error. Here we can acknowledge the error but ignore, rather than - as above - pretend it doesn't exist